scripts/host-build-all.sh

```diff
@@ -5,17 +5,59 @@
 # fail on any errors
 set -e
 
-cd tools
-cd testbench
+print_usage()
+{
+    cat <<EOFUSAGE
+usage: $0 [-f]
+       -f Build testbench with compiler provided by fuzzer
+          (default path: /home/sof/work/AFL/afl-gcc)
+EOFUSAGE
+}
 
-rm -rf build_testbench
+make_testbench()
+{
+    cd "$BUILD_TESTBENCH_DIR"
 
-mkdir build_testbench
-cd build_testbench
+    rm -rf build_testbench
 
-cmake -DCMAKE_INSTALL_PREFIX=install \
-	-DCMAKE_VERBOSE_MAKEFILE=ON \
-	..
+    mkdir build_testbench
+    cd build_testbench
 
-make -j$(nproc --all)
-make install
+    cmake -DCMAKE_INSTALL_PREFIX=install \
+	  -DCMAKE_VERBOSE_MAKEFILE=ON \
+	  ..
+
+    make -j$(nproc --all)
+    make install
+}
+
+build_with_afl()
+{
+    export CC=/home/sof/work/AFL/afl-gcc
+}
+
+main()
+{
+    SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+    SOF_REPO=$(dirname "$SCRIPT_DIR")
+    BUILD_TESTBENCH_DIR="$SOF_REPO"/tools/testbench
+    : ${NO_PROCESSORS:=$(nproc)}
+
+    DO_BUILD_with_afl=false
+
+    while getopts "fh" OPTION; do
+	case "$OPTION" in
+	    f) DO_BUILD_with_afl=true ;;
+	    h) print_usage; exit 1;;
+	    *) print_usage; exit 1;;
+	esac
+    done
+
+    if "$DO_BUILD_with_afl"; then
+	build_with_afl
+    fi
+
+    make_testbench
```

Comment:
`rebuild_testbench` because the main purpose of this script is simplify the `cmake + make` dance. `make_testbench` refers to only the second part of the dance.

Reply:
Ack. Let me fix this.